android/build.gradle.kts

```diff
@@ -13,6 +17,22 @@ android {
         versionCode = 1
         versionName = "1.0"
     }
+    signingConfigs {
+        create("release") {
+            storePassword = keystoreProperties.getProperty("KEYSTORE_PASSWORD")
+            // Remember to edit signing.properties to have the correct info for release build.
+            storeFile = file("config/release.keystore")
+            keyPassword = keystoreProperties.getProperty("KEY_PASSWORD")
+            keyAlias = keystoreProperties.getProperty("KEY_ALIAS")
+        }
+
+        getByName("debug") {
+            storeFile = file("config/debug.keystore")
+            storePassword = "jsglOe09kgjslDgpEg"
```

Comment:
Is the value sensitive? If true, please store it in a secure place.

Reply:
@phongvhd93 No. It's just a debug build. In Android Template we do this too. 🙏🏻

Reply:
@phongvhd93 For debug, it's ok to put these values here

Reply:
At least we need to put some comments here. Ex: `Replace it with your password` 🙏